llmfoundry/command_utils/data_prep/split_eval_set.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import contextlib
+import logging
+import os
+import re
+from typing import Optional
+
+import composer.utils as utils
+import numpy as np
+
+log = logging.getLogger(__name__)
+
+LOCAL_PATH = 'tmp-t'
```

Comment:
I don't think you want to hardcode this. Should this not just be a fallback to assume local path?

Reply:
im not too familiar, is it acceptable to have
```
if remote object store:
  remote object store:
else:
  local path
```

i.e. have it be the defualt?

Reply:
seems ok? just note that is what it defaults to